.travis.yml

```diff
@@ -83,15 +76,6 @@ matrix:
                CONDA_DEPENDENCIES=$CONDA_ALL_DEPENDENCIES
                PIP_DEPENDENCIES='jplephem' EVENT_TYPE='cron'
 
-        # Check for sphinx doc build warnings - we do this first because it
-        # runs for a long time. The sphinx build also has some additional
-        # dependencies. Using a more stringent locale than UTF-8.
-        - os: linux
-          env: SETUP_CMD='build_docs -w'
-               CONDA_DEPENDENCIES=$CONDA_ALL_DEPENDENCIES
-               PIP_DEPENDENCIES='sphinx-astropy jplephem pillow'
-               LC_CTYPE=C LC_ALL=C LANG=C
```

Comment:
The inclusion of the C locale is important as we've had issues in the past and this is meant as a regression test. Can you add these environment variables to the CircleCI config?

Reply:
done